tools/patch/grammar.patch

```diff
@@ -828,7 +828,7 @@ index 64e205e7fd..5bf44d6e71 100644
 +                          CHECK_NULL_ALLOWED(pegen.seq_extract_starred_exprs(this, a)),
 +                          CHECK_NULL_ALLOWED(pegen.seq_delete_starred_exprs(this, a)),
 +                          ...EXTRA) }
-+kwargs[any]:
++kwargs[KeywordOrStarredArray]:
```

Comment:
Is this because we can't use `KeywordOrStarred[]`
We could also do `KeywordOrStarred*` and then fix this up in the script to generate the parser?
Might be easier as we get more types.

(I also checked that `Array<KeywordOrStarred>` doesn't parser)

Reply:
Yep